src/controllers/pallets/PalletsForeignAssetsController.ts

```diff
@@ -0,0 +1,58 @@
+// Copyright 2017-2022 Parity Technologies (UK) Ltd.
+// This file is part of Substrate API Sidecar.
+//
+// Substrate API Sidecar is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import { ApiPromise } from '@polkadot/api';
+import { RequestHandler } from 'express';
+
+import { PalletsForeignAssetsService } from '../../services';
+import AbstractController from '../AbstractController';
+
+/**
+ * GET asset information for all foreign assets.
+ *
+ * Query:
+ * - (Optional)`at`: Block at which to retrieve runtime version information at. Block
+ *  	identifier, as the block height or block hash. Defaults to most recent block.
+ *
+ * `/pallets/foreign-assets`
+ * Returns:
+ * - `at`: Block number and hash at which the call was made.
+ * - `items`: An array containing the `AssetDetails` and `AssetMetadata` of every foreign asset.
+ *
+ * Substrate References:
+ * - Foreign Assets Pallet Instance in Kusama Asset Hub: https://github.com/paritytech/cumulus/blob/master/parachains/runtimes/assets/asset-hub-kusama/src/lib.rs#L295
+ */
+export default class PalletsForeignAssetsController extends AbstractController<PalletsForeignAssetsService> {
+	constructor(api: ApiPromise) {
+		super(api, '/pallets/foreign-assets', new PalletsForeignAssetsService(api));
+		this.initRoutes();
+	}
+
+	protected initRoutes(): void {
+		this.safeMountAsyncGetHandlers([['', this.getForeignAssets]]);
```

Comment:
```suggestion
		this.safeMountAsyncGetHandlers([['/asset-info', this.getForeignAssets]]);
```

Reply:
This needs to be added or else the only endpoint available is `/pallets/foreign-assets`

Reply:
Thank you so much @TarikGul for pointing this out. I had it before with `asset-info` but IIRC we said it is not needed (?) The main reasoning was that : 
- this endpoint shows just a list of foreign assets so `/pallets/foreign-assets` is shorter and enough to reflect the response that it gives
- also we do not have any path params like in the `/pallets/assets/{assetId}/asset-info` where we have the path param `{assetId}`

But it is no problem at all to add it again if you think it is better!